eo/CHANGELOG.md

```diff
@@ -12,17 +12,17 @@ Fixes:
 ------
 - ...
 
-0.2.3 (2022-12-13)
+0.3.0 (2022-12-20)
 ===================
 
 Changes:
 --------
 - Update base image version in Dockerfile
+- Use `mamba` instead of `conda` for environment installation
 
 Fixes:
 ------
-- na
-
+- Avoid problems with the `opencv` and `opencv-python` packages by replacing them with the `opencv-python-headless` package
```

Comment:
Since 0.2.3 was never actually working, since when I built the image, it would result in a fail. I wanted to just delete that tag and overwrite that failing attempt.

Let me know if I should still keep the tag of the failing version and changelog info.

Reply:
It's better to keep it and indicate those details in the fixed of the following release. 
Otherwise, we have to backtrack anywhere that is already deployed, which is more work and can lead to random reference errors.